apps/ledger-live-mobile/src/datadog.ts

```diff
@@ -42,3 +46,25 @@ export const initializeDatadogProvider = async (
     trackingConsent,
   });
 };
+
+/**
+ * A predicate function to determine the view name for tracking purposes.
+ *
+ * This function modifies the view name based on the current route and its parameters.
+ * If the route is the Portfolio screen, it returns null to stop tracking.
+ * For the Asset screen, it appends the currency ID to the tracked name.
+ *
+ * @param route - The current navigation route.
+ * @param trackedName - The base name to be tracked.
+ * @returns The modified view name or null if tracking should be stopped.
+ */
+export const viewNamePredicate: ViewNamePredicate = (route, trackedName) => {
+  // If the route is the Portfolio screen, we stop the native navigation tracking as we will manually track the view
```

Comment:
`const { name, params } = route;`

destructure route to reduce verbosity maybe :)